### ISSUE TITLE
Allow `nltk.corpus.wordnet` to take in PTB and Universal tags

### ISSUE THREAD

**Issue by xhluca** [nltk/nltk#2963]
WordNet (and wordnet lemmatizer) allow specifying a `pos` as an optional argument. For example:

```python
from nltk.corpus import wordnet as wn

# Synsets
print(wn.synsets("run", pos='n'))

# Lemmas
print(wn.lemmas("run", pos="v"))
```

However, many of the POS taggers either use the universal or PTB tags (e.g. `nltk.pos_tag`). If you try to input them directly here, you'd get an error:

```python
print(wn.synsets("run", pos="NN"))
# => error

print(wn.lemmas("run", pos="VERB"))
# => error
```

Right now it's possible to download `universal_tagset` and use `tagset_mapping` with some additional hardcoded mapping. However, I feel it would be more beginner friendly if you could directly give a PTB or universal tag to `wn.lemmas`, `wn.synsets`, etc.

**Comment by tomaarsen** [comment]
I think the issue arises when people are told that NLTK WordNet now supports Universal/PTB, as they might e.g. try to find synsets of a word in `VBN` (verb, past participle). That said, I recognize that allowing Universal/PTB tags is logical here, as long as we restrict this to only `NOUN`, `NN`, `VERB`, `VB` and `ADJ`, `JJ`. 

In code this simply means applying a mapping from those 6 tags I just mentioned to their corresponding WordNet variant of the pos.

**Comment by xhluca** [comment]
>  as they might e.g. try to find synsets of a word in VBN

I just checked and it seems that VB* can be lemmatized to their base form and the correct synset can be found. 

```python
# VBD
wn.synsets("ran", pos="v")
## => [Synset('run.v.01'), Synset('scat.v.01'), Synset('run.v.03'), Synset('operate.v.01'), ...]

# VBG
wn.synsets("running", pos="v")
## => [Synset('run.v.01'), Synset('scat.v.01'), Synset('run.v.03'), Synset('operate.v.01'), ...]

# VBN
wn.synsets("eaten", pos="v")
## => [Synset('eat.v.01'), Synset('eat.v.02'), Synset('feed.v.06'), Synset('eat.v.04'), ...]

# VBZ
wn.synsets("runs", pos="v")
## => [Synset('run.v.01'), Synset('scat.v.01'), Synset('run.v.03'), Synset('operate.v.01'), ...]

# VBP (e.g. "I run everyday")
wn.synsets("run", pos="v")
## => [Synset('run.v.01'), Synset('scat.v.01'), Synset('run.v.03'), Synset('operate.v.01'), ...]
```

I'm not sure if that applies to all verbs though (I don't know how the lemmatizer was built). For example, there might be exception in VBP that isn't the base form.

**Comment by xhluca** [comment]
I've started a PR towards that here: https://github.com/nltk/nltk/pull/2965

I can also add a note to the docs in that PR. Also I'm raising an `AttributeError` if the pos is incorrect, let me know if that's the correct error to raise.

**Comment by tomaarsen** [comment]
> I just checked and it seems that VB* can be lemmatized to their base form and the correct synset can be found.

True, so technically we *could* also support e.g. VBG as a "v" and go from there. That said, I'm not sure whether this would be more or less confusing. Beyond that, I delved into this a bit more, and found some issues with this approach.

First of all, `synset_from_pos_and_offset` is not the only place that relies on these `pos` values. With other words, the mapping would have to be done in more places. For example, `morphy` the lemmatizer also uses them. 
Beyond that, morphy allows you to supply a string with multiple `pos` values, such as 
```python
>>> from nltk.corpus import wordnet as wn
>>> print(wn.synsets("book", pos="vn"))
[Synset('book.v.01'), Synset('reserve.v.04'), Synset('book.v.03'), Synset('book.v.04'), Synset('book.n.01'), Synset('book.n.02'), Synset('record.n.05'), Synset('script.n.01'), Synset('ledger.n.01'), Synset('book.n.06'), Synset('book.n.07'), Synset('koran.n.01'), Synset('bible.n.01'), Synset('book.n.10'), Synset('book.n.11')]
```

In essence, this change seems like it would be more extensive than expected, and I'm not sure if it would be wise to go through with it. What are your thoughts?

**Comment by xhluca** [comment]
I'm not really sure why specifying `VBG` would conflict with specifying multiple wordnet-form pos values; do you mean the end user might get confused and try to write `VBNN`?

The PR would allow automatically converting `VB*` and `NN*` to `v` and `n`, but not the other way around. So `pos='v'` or `pos='vn'` would still work. I'm not sure I see how it would conflict with morphy as it stands. I guess an inexperienced morphy user could try to specify `pos="VB"` and get an error, but that could be cleared up in the docs by being more explicit about what `pos` values are accepted.

If you feel `synset_from_pos_and_offset` is not the right place, would you be more comfortable if the changes were done inside `wn.synsets` (and leave `synset_from_pos_and_offset` as is)? Or were you thinking about the strict mapping `NN -> n` and `VB -> v` rather than `NNP, NNPS... -> NN` and so on?
